include-build/roborazzi-core/src/commonJvmMain/kotlin/com/github/takahirom/roborazzi/RoborazziOptions.kt

```diff
@@ -38,6 +39,40 @@ fun roborazziDefaultResizeScale(): Double {
   return checkNotNull(System.getProperty("roborazzi.record.resizeScale", "1.0")).toDouble()
 }
 
+@ExperimentalRoborazziApi
+sealed interface RoborazziCaptureRoboImageFilePathStrategy {
+  val propertyValue: String
+
+  object RelativePathFromCurrentDirectory : RoborazziCaptureRoboImageFilePathStrategy {
+    override val propertyValue: String
+      get() = "RelativePathFromCurrentDirectory"
+  }
+
+  object RelativePathFromRoborazziContextOutputDirectory : RoborazziCaptureRoboImageFilePathStrategy {
+    override val propertyValue: String
+      get() = "RelativePathFromRoborazziContextOutputDirectory"
+  }
+}
+
+
+@ExperimentalRoborazziApi
+fun roborazziCaptureRoboImageFilePathStrategy(): RoborazziCaptureRoboImageFilePathStrategy {
+  return when (
+    System.getProperty(
+      "roborazzi.captureRoboImage.filePathStrategy",
```

Comment:
This property doesn't affect comparison images like hoge_compare.png. Therefore, I believe roborazzi.record.filePathStrategy would be a more appropriate name.

Reply:
And we may provide the filePathStrategy in RoborazziContext someday.